ffi/dotnet-idsdk/IoticsIdLib/src/id/IdLib.cs

```diff
@@ -0,0 +1,176 @@
+using System.Runtime.InteropServices;
+
+/**
+Wrapper class that loads the DLL and exposes the methods as C# code. 
+*/
+internal partial class IdLib
+{
+    // Define the struct that matches the return type of CreateDefaultSeed in Go
```

Comment:
1. It's for most of them, not just `CreateDefaultSeed`? 
2. `StringAndError`? not everything will match `Return`?
3. Inputs are `string` and outputs are `IntPtr` - that seems asymmetrical (but I haven't looked at the C# auto-conversions so maybe this is the best we can do)?

Reply:
3. it's due to the fact (I think) that return values are unmanaged bits of mem that need to be freed explicitly... inputs are managed by dotnet. many examples seem to follow this pattern